Task5/css/header.css

```diff
@@ -0,0 +1,107 @@
+.layout {
+    margin-top: 10px;
+    padding: 15px;
+    border: 1px solid #beb0b0;
+}
+
+body {
+    width: 1200px;
+    margin: 0 auto;
+}
+
+header {
+    height: 250px;
+    overflow: hidden;
+    position: relative;
+    z-index: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.signboard {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    background: #ffffff;
+    z-index: 1;
+    font-size: 35px;
+    font-style: italic;
+    padding: 36px 64px;
+}
+
+.signboard::before {
+    content: "";
+    position: absolute;
+    z-index: -1;
+    background: #ffffff;
+    border: 3px solid black;
+    padding: 11% 47%;
```

Comment:
`11% 47%` - what mean these magic numbers? Do they have suitable values if we will change a text?

Reply:
Check it https://i.imgur.com/I1nSV1q.png. I guess the board should be at the center of the header block, doesn't matter how many text we have inside it. Fix it! Also check the white "border" outside from the black one.

Reply:
Comments was fixed.